src/config/sysdefines.js

```diff
@@ -0,0 +1,12 @@
+export default class SysDefines {
+
+    getWebProtocol() {
+        // Replace to return "https://" for HTTPS Support
+        return `http://`;
```

Comment:
why return `http://`? Dosen‘t return "https://"?

Reply:
it will be replaced when the container run.
https://github.com/hazmi-e205/sonic-client-web/blob/6c9dd1a3543aaaedb00aaa3b3ae4528f1edc7911/replace.sh#L11